sustc-api/src/main/java/io/sustc/dto/UserRecord.java

```diff
@@ -1,44 +1,70 @@
 package io.sustc.dto;
 
-import com.opencsv.bean.CsvBindByName;
 import lombok.Data;
 
 @Data
 public class UserRecord {
 
-    @CsvBindByName(column = "Mid")
-    private Long mid;
+    /**
+     * The user's ID
+     */
+    private long mid;
 
-    @CsvBindByName(column = "Name")
+    /**
+     * The user's name
+     */
     private String name;
 
-    @CsvBindByName(column = "Sex")
+    /**
+     * The user's sex
+     */
     private String sex;
 
-    @CsvBindByName(column = "Birthday")
+    /**
+     * The user's birthday
+     */
     private String birthday;
 
-    @CsvBindByName(column = "Level")
-    private Short level;
+    /**
+     * The user's level
+     */
+    private int level;
```

Comment:
```suggestion
    private short level;
```

Reply:
I think that int or short makes no differences.

Reply:
some students may already use `smallint` to store this field, we'd make minimal changes

Reply:
ok